packages/Webkul/BookingProduct/src/Resources/assets/sass/velocity.scss

```diff
@@ -278,6 +278,19 @@ body {
                 left: 80px !important;
                 right: inherit !important;
             }
+
+            .icon {
```

Comment:
Should be placed outside the `.icon` class and wrapped around `.booking-information` class.